### DISCUSSION TITLE
AniMe Matrix support for virtual pet?

### DISCUSSION THREAD

**Discussion by ghost** [seerge/g-helper#317] [Ideas]
Is the support for this feature planned?
I use virtual pet because it's fun to have around. It used to work along with AniMe matrix, after the uninstallation of Armoury crate, I can no longer display it on AniMe matrix.

Also, any plans for audio visualizer support?

**Comment by seerge** [comment]
@flankytingz hello, I don't plan to add extra bloat to the app on this stage :)  + virtual pet is proprietary asus application that has no api and it's not possible to integrate it anywhere.

**Reply by ghost** [reply]
Okay, good to know.
